packages/create-redwood-app/README.md

```diff
@@ -85,19 +85,25 @@ The installation script is built with [Yargs](https://github.com/yargs/yargs)
 The project codebase in `template/` uses [Yarn Workspace v1](https://classic.yarnpkg.com/en/docs/workspaces/) for a monorepo project containing the API and Web Sides. Redwood packages are included in `template/package.json`, `template/web/package.json`, and `template/api/package.json`, respectively.
 
 ### How to run create-redwood-app from your local repo and create a project
-Make sure you to first run `yarn install` in your project root.
+First, run `yarn install` in the root of the mono repo. Then, run `yarn install` again, inside the `packages/create-redwood-app` directory.
```

Comment:
What makes the second yarn install necessary?

Reply:
Good catch -- you don't need a second `yarn install`, but you do need `yarn build` inside the monorepo and `yarn build` inside the package.